src/components/ColorPalette/ColorPalette.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './ColorPalette.scss';
+import { Color } from '../../types/Color';
+import { ColorItem } from '../ColorItem';
+
+const colors: Color[] = [
+  { id: 1, keyword: 'red' },
+  { id: 2, keyword: 'green' },
+  { id: 3, keyword: 'blue' },
+  // { id: 4, keyword: 'aqua' },
+  // { id: 5, keyword: 'orange' },
+];
```

Comment:
Would be nice to create separate file with such constants to manage it in separate place